pkg/prowgen/prowgen_konflux.go

```diff
@@ -96,6 +96,19 @@ func GenerateKonflux(ctx context.Context, openshiftRelease Repository, configs [
 						log.Printf("[%s] created nudges (%v) - operatorVersions: %#v - downstreamVersion: %v): %#v", r.RepositoryDirectory(), ok, operatorVersions, downstreamVersion, nudges)
 					}
 
+					prefetchDeps := konfluxgen.PrefetchDeps{}
+					if _, err := os.Stat(filepath.Join(r.RepositoryDirectory(), "rpms.lock.yaml")); err == nil {
+						// If rpms.lock.yaml is present enable dev-package-managers and RPM caching
+						prefetchDeps.DevPackageManagers = "true"
+						prefetchDeps.WithRPMs()
+					}
+					if _, err := os.Stat(filepath.Join(r.RepositoryDirectory(), "vendor")); err != nil {
```

Comment:
not sure, if os.Stat can return multiple errors, but maybe we can check if it is "FileNotExists" error (`os.IsNotExist(err)`)

Reply:
Done

Reply:
using the suggested improvement
> // This function predates errors.Is. It only supports errors returned by
// the os package. New code should use errors.Is(err, fs.ErrNotExist).